<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureFilesystem</class>
 <widget class="QWidget" name="ConfigureFilesystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Storage Directories</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0"><widget class="QLabel" name="label"><property name="text"><string>NAND</string></property></widget></item>
      <item row="0" column="2"><widget class="QLineEdit" name="nand_directory_edit"/></item>
      <item row="0" column="3"><widget class="QToolButton" name="nand_directory_button"><property name="text"><string>...</string></property></widget></item>
      <item row="1" column="0"><widget class="QLabel" name="label_2"><property name="text"><string>SD Card</string></property></widget></item>
      <item row="1" column="2"><widget class="QLineEdit" name="sdmc_directory_edit"/></item>
      <item row="1" column="3"><widget class="QToolButton" name="sdmc_directory_button"><property name="text"><string>...</string></property></widget></item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title"><string>Gamecard</string></property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1"><widget class="QCheckBox" name="gamecard_inserted"><property name="text"><string>Inserted</string></property></widget></item>
      <item row="1" column="1"><widget class="QCheckBox" name="gamecard_current_game"><property name="text"><string>Current Game</string></property></widget></item>
      <item row="2" column="1"><widget class="QLabel" name="label_3"><property name="text"><string>Path</string></property></widget></item>
      <item row="2" column="2"><widget class="QLineEdit" name="gamecard_path_edit"/></item>
      <item row="2" column="3"><widget class="QToolButton" name="gamecard_path_button"><property name="text"><string>...</string></property></widget></item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title"><string>Patch Manager</string></property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0"><widget class="QLabel" name="label_8"><property name="text"><string>Dump Root</string></property></widget></item>
      <item row="0" column="2"><widget class="QLineEdit" name="dump_path_edit"/></item>
      <item row="0" column="3"><widget class="QToolButton" name="dump_path_button"><property name="text"><string>...</string></property></widget></item>
      <item row="1" column="0"><widget class="QLabel" name="label_9"><property name="text"><string>Mod Load Root</string></property></widget></item>
      <item row="1" column="2"><widget class="QLineEdit" name="load_path_edit"/></item>
      <item row="1" column="3"><widget class="QToolButton" name="load_path_button"><property name="text"><string>...</string></property></widget></item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item><widget class="QCheckBox" name="dump_nso"><property name="text"><string>Dump Decompressed NSOs</string></property></widget></item>
        <item><widget class="QCheckBox" name="dump_exefs"><property name="text"><string>Dump ExeFS</string></property></widget></item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="autoloader_group">
     <property name="title"><string>Autoloader</string></property>
     <layout class="QVBoxLayout">
      <item><widget class="QCheckBox" name="prompt_for_autoloader"><property name="text"><string>Prompt to run Autoloader when a new game directory is added</string></property></widget></item>
      <item><widget class="QPushButton" name="run_autoloader_button"><property name="text"><string>Run Autoloader Now</string></property></widget></item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="updater_group">
     <property name="title"><string>Updater</string></property>
     <layout class="QGridLayout" name="gridLayout_updater">
      <item row="0" column="0" colspan="2"><widget class="QCheckBox" name="enable_backups_checkbox"><property name="text"><string>Enable AppImage Backups</string></property></widget></item>
      <item row="1" column="0"><widget class="QCheckBox" name="custom_backup_location_checkbox"><property name="text"><string>Use Custom Backup Location</string></property></widget></item>
      <item row="2" column="0"><widget class="QLineEdit" name="custom_backup_location_edit"/></item>
      <item row="2" column="1"><widget class="QToolButton" name="custom_backup_location_button"><property name="text"><string>...</string></property></widget></item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title"><string>Caching &amp; Scanning</string></property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0"><widget class="QCheckBox" name="cache_game_list"><property name="text"><string>Cache Game List Metadata</string></property></widget></item>
      <item row="0" column="1"><widget class="QPushButton" name="reset_game_list_cache"><property name="text"><string>Reset Metadata Cache</string></property></widget></item>
      <item row="1" column="0" colspan="2"><widget class="QCheckBox" name="scan_nca"><property name="text"><string>Scan for .nca files (Advanced: Significantly slows down scanning)</string></property></widget></item>
     </layout>
    </widget>
   </item>
   <item><spacer name="verticalSpacer"><property name="orientation"><enum>Qt::Vertical</enum></property></spacer></item>
  </layout>
 </widget>
</ui>
